backend/src/apiserver/server/experiment_server.go

```diff
@@ -44,7 +46,23 @@ func (s *ExperimentServer) ListExperiment(ctx context.Context, request *api.List
 	if request.PageToken != "" {
 		opts, err = list.NewOptionsFromToken(request.PageToken, int(request.PageSize))
 	} else {
-		opts, err = list.NewOptions(&model.Experiment{}, int(request.PageSize), request.SortBy, request.Filter)
+		var f *api.Filter
```

Comment:
Any chance we can reuse this code instead of copying it?

Reply:
Thanks! I consolidated this logic into a new function, and also added tests for it. PTAL.